service-catalog/systems/service-catalog.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: backstage.io/v1alpha1
+kind: System
```

Comment:
This system is missing postgres crunchy operator Component

Reply:
Resolved in the latest commit.